tools/n64tool.c

```diff
@@ -23,7 +23,7 @@
 #include <stdlib.h>
 #include <string.h>
 
-#define WRITE_SIZE	(1024*1024)
+#define WRITE_SIZE	1024

```

Comment:
Is there a reason to reduce this?

Reply:
This is also revering my change, and thus introducing a performance regression.